code_generation/requirements.txt.license

```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model project <dynamic.grid.calculation@alliander.com>
+
```

Comment:
Nitish already add these two in the dev dependency. We don't need this file here.